CHANGELOG.md

```diff
@@ -0,0 +1,95 @@
+# Install or Defer Change Log
+
+All notable changes to this project will be documented in this file. This project adheres to [Semantic Versioning](http://semver.org/).
+
+
+## [Unreleased] - TBD
+
+### Added
+
+- added `--no-scan` flag to `--download` and `--install` commands (avoids repeatedly checking for updates after initial `softwareupdate --list`, speeding up script runtime)
+
+
+## [2.1.2] - 2019-04-11
```

Comment:
May want to change the release dates to when those commits occurred rather than when we compiled the packages and built the GitHub releases, so it's clearer when the script changes were made.